### DISCUSSION TITLE
Can i use Web3Auth without the modal?

### DISCUSSION THREAD

**Discussion by YZhenY** [Web3Auth/Web3Auth#337] [Help]
Love the logins, but don't want to use the standard Web3Auth modal, can I?

**Comment by yashovardhan** [comment]
Yes, you can directly use the Web3Auth Plug and Play Core SDK (https://web3auth.io/docs/sdk/web/core/) and build your own custom UI using Web3Auth working in the backend. The Web3Auth Modal SDK is just built on top of this so all the functions will be supported properly.

The major difference comes into play while logging in the user. Make sure to change your logins according to the `connectTo` function. https://web3auth.io/docs/sdk/web/core/usage#logging-in-the-user

**Reply by briandavidwetzel** [reply]
@yashovardhan, Is this also possible with the React Native SDK? I have not seen any documentation for this or code samples.

**Comment by NTonani** [comment]
@yashovardhan @YZhenY The accepted answer doesn't specify _how_ to "change your logins according to the `connectTo`". Can more details be provided please? We have been hitting walls.

**Reply by NTonani** [reply]
We tried following: https://web3auth.io/docs/custom-authentication/byo-jwt-providers, which suggests we can directly provide a JWT via `id_token`. [ExtraLoginOptions](https://web3auth.io/docs/sdk/web/openlogin#extraloginoptions) does not provide this as an accepted field though, and I'm unclear how it circumvents the `uxMode` of the adapter.
